app/jobs/regular/run_problem_check.rb

```diff
@@ -15,14 +15,8 @@ def execute(args)
 
       check = ProblemCheck[identifier]
 
-      problems = check.call
-      raise RetrySignal if problems.present? && retry_count < check.max_retries
-
-      if problems.present?
-        problems.each { |problem| AdminDashboardData.add_found_scheduled_check_problem(problem) }
-        ProblemCheckTracker[identifier].problem!(next_run_at: check.perform_every.from_now)
-      else
-        ProblemCheckTracker[identifier].no_problem!(next_run_at: check.perform_every.from_now)
+      check.run do |problems|
+        raise RetrySignal if problems.present? && retry_count < check.max_retries
       end
```

Comment:
`ProblemCheck#run` takes a block as an escape hatch before updating the trackers. This allows us, in this case, to short-circuit and trigger a retry without registering a "blip".